cognite/neat/rules/exporters/_rules2ontology.py

```diff
@@ -324,12 +325,14 @@ def from_list_of_properties(cls, definitions: list[InformationProperty], namespa
         )
         for definition in definitions:
             owl_property.type_.add(OWL[definition.type_])
-            owl_property.range_.add(
-                XSD[definition.value_type.suffix]
-                if definition.value_type.suffix in XSD_VALUE_TYPE_MAPPINGS
-                else namespace[definition.value_type.suffix]
-            )
-            owl_property.domain.add(namespace[definition.class_.suffix])
+
+            if isinstance(definition.value_type, DataType):
+                owl_property.range_.add(XSD[definition.value_type.xsd])
```

Comment:
This is causing issue since for example if we have `String` data_type, corresponding `.xsd` attribute returns `xsd:strign` which then when passed to `XSD` namespace creates URI `http://www.w3.org/2001/XMLSchema#xsd:string` which is ill formed URI.  Instead this method is suppose to generate `http://www.w3.org/2001/XMLSchema#string`

We have two options, either remove `xsd:` all toghether from all `data_type` definitions, or do XSD[definition.value_type.xsd.removeprefix("xsd:"). I think we should go with the first one (as just discussed in the call).